requirements/base.txt

```diff
@@ -23,7 +23,7 @@ pyopenssl~=23.2.0
 jsonschema<4.18
 
 # Needed for supporting Protocol in Python 3.7, Protocol class became public with python3.8
-typing_extensions~=4.4.0
+typing_extensions>=4.4.0
```

Comment:
Can we also add `<5`? usually we want to be more cautious about major version upgrade.

```suggestion
typing_extensions>=4.4.0,<5
```
```